bpfd-operator/pkg/helpers/helpers.go

```diff
@@ -169,13 +171,43 @@ func GetMaps(c *bpfdclientset.Clientset, ProgramName string, mapNames []string)
 
 	prog := bpfProgramList.Items[0]
 
+	id, ok := prog.Annotations[internal.IdAnnotation]
+	if !ok {
+		return nil, fmt.Errorf("BpfProgram %s does not have a program id", ProgramName)
+	}
+
+	maps := map[string]string{}
+	programMapPath := fmt.Sprintf("%s/%s", internal.BpfdMapFs, id)
```

Comment:
This doesn't work if this program is sharing another program's maps.

Reply:
good point... how did that work before though?

Reply:
I guess it didn't.

Reply:
Blocked by #744. Once ProgramData is written to annotations, the MapPinPath will be there and the path doesn't need to be calculated.